core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1358,6 +1378,17 @@ oppia.directive('schemaBasedFloatEditor', [function() {
           }
         };
 
+        if (typeof $scope.uiConfig() !== 'undefined' &&
```

Comment:
This is to avoid errors where slider is not used. Otherwise everywhere the float schema is used it should have a ui_config with a slider attribute defined.